src/types/Instance.ts

```diff
@@ -1,11 +1,10 @@
 import { IRow } from "@patternfly/react-table";
 
 export interface Instance extends IRow {
-  id: string;
-  endpoint: string;
-  name: string;
-  owner: string;
-  published_at: string;
-  status: string;
-  submitted_at: string;
+  id?: string;
+  endpoint?: string;
+  name?: string;
+  published_at?: string;
+  status?: string;
+  submitted_at?: string;
```

Comment:
This is just a curiosity. Is this interface still needed or could we use the generated one?

Reply:
Removed both interface and details action